### PR TITLE
Attempt to fix broken envs during build

### PR DESCRIPTION
**Pull request by Frizi**
### Pull Request Description

Attempt to fix build env issues causing "Enso undefined" version and missing AG grid license key.

Screenshot taken from a CI build:
image

### CODE REVIEW

File: app/common/src/appConfig.js [hunk 1]
```diff
@@ -67,6 +79,16 @@ export async function readEnvironmentFromFile() {
   }
 }
 
+/**
+ * Discard environment variable value when is an "undefined" string.
+ * @param {string} name Name of an env variable.
+ */
+function discardUndefinedEnv(name) {
```
Comment (vitvakatu) [inline app/common/src/appConfig.js:71]: I don’t see it being used anywhere?

Comment (farmaazon) [reply app/common/src/appConfig.js:71]: > I don’t see it being used anywhere?

Line 29

File: build-config.yaml [hunk 2]
```diff
@@ -6,4 +6,4 @@ required-versions:
   # NB. The Rust version is pinned in rust-toolchain.toml.
   # NB. The Node version is pinned in .node-version.
   cargo-watch: ^8.1.1
-  wasm-pack: ^0.12.1
+  wasm-pack: ">=0.12.1"
```
Comment (farmaazon) [inline build-config.yaml:9]: Why this change? Are not afraid of wasm-pack updates even a bit?

**Review by farmaazon — APPROVED** [review]

File: build-config.yaml (hunk 2 above)
Comment (Frizi) [reply build-config.yaml:9]: I am using 0.13 and there is no issue with that. Historically they also didn't really make much updates. 0.13 was released a few weeks ago and it was a first release since a few years back. For CI, we pin the installed version anyway. This change just makes the potential compatibility issues with local setup go away, as any new installation will default to 0.13 now. We are phasing out wasm-pack usage completely soon enough anyway.

**Comment by MrFlashAccount** [comment]
I added the labels above as possible solution for failed storybook deploy task: https://github.com/enso-org/enso/actions/runs/11953657884/job/33322022010?pr=11584 

For some reason it doesn't detect the chromatic config which happen on some PRs but doesn't on the others

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/enso-org/enso">Attempt to fix broken envs during build · enso-org/enso@295e155</a></strong></div><div>Enso Analytics is a self-service data prep and analysis platform designed for data teams. - Attempt to fix broken envs during build · enso-org/enso@295e155</div></blockquote>

**Review by MrFlashAccount — APPROVED** [review]

File: app/common/src/appConfig.js (hunk 1 above)
Comment (vitvakatu) [reply app/common/src/appConfig.js:71]: Oh, indeed, somehow it didn’t show up in search
